Sources/CariocaIndicatorView.swift

```diff
@@ -39,7 +39,7 @@ public class CariocaIndicatorView: UIView {
 	///The custom indicator configuration
 	private let config: CariocaIndicator
 	///The constraints applied to the iconview. Can be updated later with custom configuration
-	private var iconConstraints: [NSLayoutConstraint] = []
+	private var iconConstraints: EdgeConstraints!
```

Comment:
Is there a way we can avoid this force-unwrap ?

Reply:
Well, take a look at `init(edge: UIRectEdge, indicator: CariocaIndicator)`, only if we can add (or, at least create) constraints before calling super.init(frame: frame).

We can make it `lazy var` and call after `init` is completed, but it sounds messy.

Reply:
right, then keep it as it is :)